src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -41,109 +41,112 @@
  * com.snowflake.kafka.connector.internal.streaming.SnowflakeSinkServiceV2}
  */
 public class SnowflakeSinkTaskForStreamingIT {
-
+  private int taskId;
+  private int partitionCount;
   private String topicName;
-  private static int partition = 0;
-  private TopicPartition topicPartition;
   private Map<String, String> config;
-
-  @Mock Logger logger = Mockito.mock(Logger.class);
```

Comment:
these are only used for the logging test, so moved down there to increase readability

Reply:
moved to a following pr, so this comment is out of context